provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -167,9 +150,9 @@ def _calculate_allowed_failure(self, layout: Layout) -> AllowedFailures:
         # This is a workaround for getting cvg by looking at one data node
         # only. The implementation here needs to be corrected using a
         # different task (EOS-27063).
-        data_nodes = conf.get_data_nodes()
-        node_count = len(data_nodes)
-        machine_id = data_nodes[0]
+        nodes = conf.get_machine_ids_for_service(Const.SERVICE_MOTR_IO.value)
```

Comment:
Hi @rahul27kumar 
Variable name data_nodes need not be changed here as it is specifically for data_nodes and not all the nodes.